### PR TITLE
NASA Media Gallery

### PR DESCRIPTION
**Pull request by SonaliKumari2**
### Issue Title: NASA Media Gallery

- **Name of the project** : NASA Media Gallery
- **Category:**  CSS, Front-end,  Vanilla JS, Documentation.
- **Name:** Sonali Kumari
- **GitHub ID:** SonaliKumari2
- **Idenitfy yourself:** Contributor

Closes: #116 

### Describe the add-ons or changes you've made 📃
NASA API to fetch images based on the topic entered by the user from NASA Gallery

Give a clear description of what have you added or modifications made

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

### How Has This Been Tested? ⚙️
By running index.html .

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### Screenshots 📷

Screenshot 2024-05-21 134144

### Note to reviewers 📄
Review.

### CODE REVIEW

File: Front-end-Projects/Intermediate/Netflix-Clone/README.md
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Netflix-Clone/README.md:1]: Why this is edited?

File: Vanilla-JS-Projects/Basic/NASA-Media-Gallery/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<h1 align='center'><b>💥 TO DO LIST 💥</b></h1>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/NASA-Media-Gallery/README.md:1]: carefully write the README file

File: Vanilla-JS-Projects/Basic/NASA-Media-Gallery/screenshot.webp/ss1.webp
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/NASA-Media-Gallery/screenshot.webp/ss1.webp:1]: there is not requirement of directory named `screenshot.webp`
I think you have to go through other merged prs of type `Vanilla-JS-Projects`.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@SonaliKumari2 Resolve these changes and join the discord server for further communication.

**Comment by SonaliKumari2** [comment]
Yes, I am working on it. As I am a beginner, it is taking me time

File: Vanilla-JS-Projects/Basic/NASA-Media-Gallery/screenshot.webp/ss1.webp
Comment (SonaliKumari2) [reply Vanilla-JS-Projects/Basic/NASA-Media-Gallery/screenshot.webp/ss1.webp:1]: what changes should i made in my readme.md file?

**Comment by Avdhesh-Varshney** [comment]
@SonaliKumari2 again raise your pr by re-forking this project. It can't be approved.

**Pull request closed without merging**